airbyte-webapp/src/components/EntityTable/components/StreamStatusIcons/Checkmark.tsx

```diff
@@ -0,0 +1,9 @@
+export const Checkmark = () => (
+  <svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I always assumed that this xmlns="http://www.w3.org/2000/svg" is removed when the svg is embedded into html.